Dockerfile

```diff
@@ -9,21 +9,14 @@ ARG CONTAINER_USERID
 # Configure our user
 RUN usermod -u $CONTAINER_USERID frontend
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
-RUN chown -R frontend /obs/src/api
-
 # Now do the rest as the user with the same ID as the user who
 # builds this container
 USER frontend
-WORKDIR /obs/src/api
 # foreman, which we only run in docker, needs a different thor version than OBS.
 # Installing the gem directly spares us from having to rpm package two different thor versions.
-RUN gem install thor:0.19 foreman
-
-# FIXME: Retrying bundler if it fails is a workaround for https://github.com/moby/moby/issues/783
-#        which seems to happen on openSUSE (< Tumbleweed 20171001)...
-RUN export NOKOGIRI_USE_SYSTEM_LIBRARIES=1; bundle install --jobs=3 --retry=3 || bundle install --jobs=3 --retry=3
+RUN sudo gem.ruby2.5 install thor:0.19 foreman
```

Comment:
💭  Didn't you add a Gemfile in another pull request?

Reply:
The mysql2 gem has to be installed as rpm to cover all dependencies. Otherwise we would have to install those other packages explicitly.
So we would have had a Gemfile with a single gem. I didn't see the point in that.

Reply:
Ah okay...